linux_os/guide/services/ntp/chronyd_no_chronyc_network/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable network management of chrony daemon'
+
+description: |-
+    The <tt>cmdport</tt> option in <tt>/etc/chrony.conf</tt> can be set to
+    <tt>0</tt> to stop chrony daemon from listening on the UDP port 323
+    for management connections made by chronyc.
+
+rationale: |-
+    Not exposing the management interface of the chrony daemon on
+    the network diminishes the attack space.
+
+severity: unknown
```

Comment:
What severity should this rule be assigned?

Reply:
@stevegrubb, what severity is this?